game_template/README.md

```diff
@@ -0,0 +1,538 @@
+A tiny game in Flutter with all the bells and whistles
+of a mobile (iOS & Android) game:
+
+- sound
+- music
+- main menu screen
+- settings
+- ads (AdMob)
+- in-app purchases
+- games services (Game Center & Google Play Games Services)
+- crash reporting (Firebase Crashlytics)
+
+
+# Getting started
+
+The game compiles and works out of the box. It comes with things
+like a main menu, a router, a settings screen, and audio.
+When building a new game, that is likely everything you will need at first.
+
+When you're ready to enable the more advanced integrations, like ads
+and in-app payments, read the _Integrations_ section below.
+
+
+# Development
+
+To run the app in debug mode:
+
+    flutter run
+
+This assumes you have an Android emulator, iOS Simulator or a physical
+device attached.
+
+It is often convenient to develop your game as a desktop app.
+For example, you can run `flutter run -d macOS`, and get the same UI
+in a desktop window on a Mac. That way, you don't need to be running
+Simulators nor attaching mobile devices. This template supports that
+method of development: integrations like AdMob will be disabled then.
+
+
+## Code organization
+
+Code is organized in a loose and shallow feature-first fashion.
+In `lib/src`, you'll therefore find directories such as `ads`, `audio`
+or `main_menu`. Nothing fancy, but usable.
+
+```
+lib
+├── src
+│   ├── ads
+│   ├── app_lifecycle
+│   ├── audio
+│   ├── crashlytics
+│   ├── game_internals
+│   ├── games_services
+│   ├── in_app_purchase
+│   ├── level_selection
+│   ├── main_menu
+│   ├── play_session
+│   ├── player_progress
+│   ├── settings
+│   ├── style
+│   └── win_game
+├── ...
+└── main.dart
+```
+
+The state management approach is intentionally low-level. That way, it's easy to
+take this project and run with it, without having to learn new paradigms, or having
+to remember to run `flutter pub run build_runner watch`. You are,
+of course, encouraged to use whatever paradigm, helper package or code generation
+scheme to build on top of this project.
+
+
+## Building for production
+
+To build the app for iOS (and open Xcode when finished):
+
+```bash
+flutter build ipa && open build/ios/archive/Runner.xcarchive
+```
+
+To build the app for Android (and open the folder with the bundle when finished):
+
+```bash
+flutter build appbundle && open build/app/outputs/bundle/release
+```
+
+While the template is meant for mobile games, you can also publish
```

Comment:
As a reader this comment makes me think the sample while not intended for web will at least be buildable/runnable. I can see confusion popping up when the sample won't work. 

Perhaps we can clarify that is not the case for this specific sample or that more changes would be necessary?

Reply:
Ah, thanks for mentioning it. The game _is_ runnable on the web, but I had a bug with the Crashlytics code that would crash the game on startup in the browser.

It's fixed now.

Reply:
Even better, thanks!